ep001/README.md

```diff
@@ -0,0 +1,50 @@
+# [python cli tested with pytest - (beginner to intermediate)](https://youtu.be/qvkppppy9K8?list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY)
```

Comment:
I'd drop the playlist id from this

Reply:
Ok, I've got the link from the projects README.md

Reply:
npnp, can just remove everything after `?`